README.md

```diff
@@ -20,15 +19,16 @@ The entire configuration of the site is done via a json config file. The configu
 
 
 ## Connectors
-Connectors are what Jane uses to pull in information, interpret them and issue out a response. The Routes specify where the results from the input should be written to or * for all. The Target can specify a channel in the case of slack. To add a new connector, Put them in the connectors folder and make an entry in connectors/list.go.
+Connectors are what Jane uses to pull in information, interpret it, and issue out a response. The Routes specify where the results from the input should be written to or * for all. The Target can specify a channel in the case of Slack. 
```

Comment:
Good call, stuff like that we should specify in each connector section. Want to write one for Twilio?